### PR TITLE
Nav Link Alignment

### LINKED ISSUE #148: Link Alignment Fix
## Expected Behavior

_Report what **should** be happening._
Button text should be centered on desktop.

## Actual/Current Behavior
Button text is centered for mobile devices but not desktop sized screens.

_Describe what is currently happening._
Button text is not centered on desktop screens.

## Steps to Reproduce the Problem

  1.  Open application and view button.

## Pre-Testing TODOs

_What needs to be done before testing._

  1. Adjust CSS to center button text

## Testing Steps

Open app and observe button.

_How do the users test this change?_

  1. Navigate to home screen
  2. Observe "All Pokemon" button

## Environment

Desktop browsers

### PR DESCRIPTION
**Pull request by Gjhnsn**
## Changes
1. Adjust link alignment in header for desktop.
2. Fixed offset alignments on mobile screens due to the desktop adjustment.

## Purpose
To center button text.

## Approach
Using SCSS to adjust position of button text.

Closes #148

### CODE REVIEW

**Review by Lap343 — APPROVED** [review]
Lookin Good! No complaints here

File: src/styles/components/_header.scss [hunk 1]
```diff
@@ -189,7 +190,7 @@ input[type="text"]:focus {
 
   .nav-link {
     padding-bottom: 8px;
-    padding-top: 0px;
+    margin-top: 0;
```
Comment (krferrell) [inline src/styles/components/_header.scss:193]: Does the margin-top need to have a value of 0? Can you just leave it off?

**Review by dannysanchez559 — APPROVED** [review]

**Review by krferrell — APPROVED** [review]
Nice work. I left a comment about the margin.

**Review by BlackCubes — APPROVED** [review]
Good styling choices!

File: src/styles/components/_header.scss (hunk 1 above)
Comment (kmausolf) [reply src/styles/components/_header.scss:193]: Might want to override the parent's properties, perhaps?

Comment (Gjhnsn) [reply src/styles/components/_header.scss:193]: Yeah it does, the button size at this point is getting smaller so without specifying 0 it throws the alignment off.

**Pull request merged**